web_transport/sdk/impl/http3_server_session.cc

```diff
@@ -15,6 +15,7 @@
 #include "impl/http3_server_session.h"
 #include "impl/http3_server_stream.h"
 #include "net/third_party/quiche/src/quic/core/http/quic_server_initiated_spdy_stream.h"
+#include "base/debug/stack_trace.h"
```

Comment:
why adding this header?

Reply:
Removed. This header was added for debugging.